www/front_src/src/Resources/index.tsx

```diff
@@ -117,6 +118,19 @@ const Resources = (): JSX.Element => {
     setSearch(searchFieldValue);
   };
 
+  const rowColorConditions = [
+    {
+      name: 'inDowntime',
+      condition: ({ in_downtime }): boolean => in_downtime,
+      color: purple[500],
+    },
+    {
+      name: 'acknowledged',
+      condition: ({ acknowledged }): boolean => acknowledged,
+      color: lime[900],
```

Comment:
Since acknowledgement color will be used in other places, should we add this color to themeProvider ?

Reply:
It will be handled in the next PR